### PR TITLE
[Doc] Add basic htaccess config for simple setup

### PR DESCRIPTION
**Pull request by andrerom**
Pulled in a few tricks from https://raw.githubusercontent.com/symfony/symfony-standard/2.8/web/.htaccess

### CODE REVIEW

**Comment by emodric** [comment]
Why not make it more inline with vhost config, with environment variables and rewrite rules that check for dev/prod environment and whatnot?

**Comment by andrerom** [comment]
Cause I was thinking of just placing it in web folder as a default fallback, hence it would not be something you edit normally it would just be there. But as that might cause lots of issues for those that forget `AllowOverride None` in vhost setup.

File: INSTALL.md [hunk 1]
```diff
@@ -114,10 +114,23 @@
...
+3. **Configure a Webserver**:
 
-    A virtual host setup is the recommended, most secure setup of eZ Publish.
-    General virtual host setup template for Apache and Nginx can be found in [doc/ folder](doc/).
+    eZ Platform can theoretically run with any web server cable of running PHP, but for most secure operation
+    we provide ready made configuration templates for the following servers:
+
+    A. **Apache**
+
+    Recommended setup for prod setup is using a virtual host setup, see [doc/apache2 folder](doc/apache2/) for details.
```
Comment (bdunogier) [inline INSTALL.md]: That's a lot of "setup". What about "A virtual host based setup is recommended for production. See..."

File: INSTALL.md [hunk 2]
```diff
@@ -114,10 +114,23 @@
...
+
+    Tip: For just testing eZ Platform you may copy provided .htaccess file into web folder:
+    ```bash
+    $ cp doc/apache2/.htaccess web/
+    ```
+
+    B. **Nginx**
+
+    Recommended setup for prod setup is using a virtual host setup, see [doc/nginx folder](doc/nginx/) for details.
```
Comment (bdunogier) [inline INSTALL.md]: Same as above.

**Comment by bdunogier** [comment]
If somebody has AllowOverride disabled, will the .htaccess file will be ignored ? Or will it throw an error ?

**Comment by emodric** [comment]
> Cause I was thinking of just placing it in web folder as a default fallback, hence it would not be something you edit normally it would just be there. But as that might cause lots of issues for those that forget AllowOverride None in vhost setup, I didn't. pov?

Hm... I wouldn't want to keep it in `web` folder for sure. Doc is enough. IIRC, it was like that for legacy too, vhost variant of apache setup and .htaccess variant of apache setup.

**Comment by andrerom** [comment]
> If somebody has AllowOverride disabled, will the .htaccess file will be ignored ?

Afaik `AllowOverride None` means Apache won't look for it at all, so no lookup, and no conflict.

**Comment by emodric** [comment]
Most of the time .htaccess (if used of course) will need to be edited, because you need to open up additional stuff to the world (google domain verifications and so on). Often .htaccess files differ in dev and prod environment, so having a basic .htaccess file which will not be changed is not desired.

IMO, an .htaccess file that completely matches & mirrors vhost setup should be provided in the docs and that's it (the way it is now in this PR).

**Comment by andrerom** [comment]
Closing, will instead move this inline in readme for apache config as a simple example for prod, to avoid having to cover everything.

**Pull request closed without merging**